app/metrics.yaml

```diff
@@ -8015,19 +7903,6 @@ recently_visited_homepage:
     data_sensitivity:
       - interaction
     expires: 114
-  search_group_opened:
```

Comment:
If this is also used when opening a group from "recently visisted" which navigates to history, then we need to keep it.

Reply:
✅